python-magic/meta.yaml

```diff
@@ -0,0 +1,59 @@
+package:
+  name: python-magic
+  version: !!str 0.4.6
```

Comment:
Why this version with the `!!`? Probably I am missing something in the yaml specification capabilities... or something else... so asking you to learn a little bit...